src/index.ts

```diff
@@ -1 +1,18 @@
-export { default } from './Sdk';
+export { default } from './Sdk/Sdk';
+export {
+    Contact,
+    Coverage,
+    Entity,
+    Pagination,
+    RoomRef,
+    Story,
+} from './types'
```

Comment:
```suggestion
} from './types';
```

The build does not fail - it looks like we're not enforcing `prettier` in this project in any way, even though we have `.prettierrc` and appropriate script in `package.json`.